net/keepalived/files/usr/bin/keepalived-rsync-inotify

```diff
@@ -0,0 +1,27 @@
+#/bin/sh
+
+[ $# -lt 3 ] && exit 1
+
+vrrp_instance=$1
+peer=$2
+rsync_dir=$3
+
+INOTIFY_ACTIONS="create delete modify move moved_to moved_from"
+ARGS=${INOTIFY_ACTIONS:+ -e ${INOTIFY_ACTIONS// / -e }}
+
+fifo=/tmp/inotifywait.$$
```

Comment:
Can we change the name. When I read this in the system, I don't know what it's for.
Maybe move this into a subdirectory /tmp/keepalived/ ?

Reply:
updated